lib/helpers.ts

```diff
@@ -217,3 +219,8 @@ export function fill(value: string, times: number): string[]{
 
 	return repeatedValues;
 }
+
+export function defaultProfileDir(): string {
+	var cacheRootDirectoryPath = hostInfo.isWindows() ? process.env.LocalAppData : path.join(osenv.home(), ".local/share");
+	return path.join(cacheRootDirectoryPath, "Telerik", "BlackDragon", ".appbuilder-cli");
```

Comment:
Actually we can. But this means that we need to move cache directory again. If this is ok, I'll do it.